cfme/cloud/tenant.py

```diff
@@ -254,6 +306,8 @@ def wait_for_appear(self, timeout=600):
                         message='Wait for cloud tenant to appear', fail_func=self.browser.refresh)
 
     def update(self, updates, wait=True):
+        if not self.exists:
```

Comment:
**REQUIRED**: No, we shouldn't put this here, if it goes anywhere, it goes at the end of the create.